src/modules/Checkbox/Checkbox.js

```diff
@@ -250,6 +237,7 @@ export default class Checkbox extends Component {
           {...htmlInputProps}
           checked={checked}
           className='hidden'
+          disabled={disabled}
```

Comment:
This is the actual change, passing disabled attribute to the input if set as a prop to Checkbox